src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+class Circle extends Figure {
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+    private double radius;
```

Comment:
Declare class fields at the beginning of the class, before any methods or constructors.